.idea/dictionaries/ddp.xml

```diff
@@ -0,0 +1,15 @@
+<component name="ProjectDictionaryState">
```

Comment:
Looks like you accidentally committed these files - you could add `.idea/*` to the `.gitignore` file in the project root directory to ignore files like this in the future.